__tests__/unit/tagged-format.test.ts

```diff
@@ -142,143 +143,170 @@ describe.each`
-          coll: collection_mod,
-          name: "Bugs",
-        }),
-        namedDoc: new NamedDocument({
-          coll: collection_mod,
-          name: "Bugs",
-          ts: TimeStub.from("2023-03-20T00:00:00Z"),
-        }),
-        nullDoc: new NullDocument(
-          new DocumentReference({ coll: bugs_mod, id: "123" }),
-          "not found"
-        ),
-        // Set types
-        // TODO: uncomment to add test once core accepts `@set` tagged values
-        // page: new Page({ data: ["a", "b"] }),
-        // TODO: uncomment to add test once core accepts `@set` tagged values
-        // page_string: new Page({ after: "abc123" }),
-      })
-    );
-
-    const backToObj = JSON.parse(result)["@object"];
-
-    // literals
-    expect(backToObj.double).toStrictEqual({ "@double": "4.14" });
-    expect(backToObj.null).toBeNull();
-    // objects and arrays
-    expect(backToObj.child.more.itsworking).toStrictEqual({
-      "@date": "1983-04-15",
-    });
-    expect(backToObj.extra).toHaveLength(2);
-    // Document types
-    expect(backToObj.mod).toStrictEqual({ "@mod": "Bugs" });
-    expect(backToObj.docReference).toStrictEqual({
-      "@ref": { coll: { "@mod": "Bugs" }, id: "123" },
-    });
-    expect(backToObj.doc).toStrictEqual({
-      "@ref": { coll: { "@mod": "Bugs" }, id: "123" },
-    });
-    expect(backToObj.namedDocReference).toStrictEqual({
-      "@ref": { coll: { "@mod": "Collection" }, name: "Bugs" },
-    });
-    expect(backToObj.namedDoc).toStrictEqual({
-      "@ref": { coll: { "@mod": "Collection" }, name: "Bugs" },
-    });
-    expect(backToObj.nullDoc).toStrictEqual({
-      "@ref": { coll: { "@mod": "Bugs" }, id: "123" },
+        mod: { value: { "@mod": "Bugs" } },
+        docReference: {
+          value: {
+            "@ref": { coll: { "@mod": "Bugs" }, id: "123" },
+          },
+        },
+        doc: {
+          value: {
+            "@ref": { coll: { "@mod": "Bugs" }, id: "123" },
+          },
+        },
+        namedDocReference: {
+          value: {
+            "@ref": { coll: { "@mod": "Collection" }, name: "Bugs" },
+          },
+        },
+        namedDoc: {
+          value: {
+            "@ref": { coll: { "@mod": "Collection" }, name: "Bugs" },
+          },
+        },
+        nullDoc: {
+          value: {
+            "@ref": { coll: { "@mod": "Bugs" }, id: "123" },
+          },
+        },
+      },
+      // Set types
+      // TODO: expect set types to be encoded as `@set` tagged values
     });
-    // Set types
-    // TODO: uncomment to add test once core accepts `@set` tagged values
-    // expect(backToObj.page).toStrictEqual({ "@set": { data: ["a", "b"] } });
-    // TODO: uncomment to add test once core accepts `@set` tagged values
-    // expect(backToObj.page_string).toStrictEqual({ "@set": "abc123" });
   });
 
   it("handles conflicts", () => {
-    const result = TaggedTypeFormat.encode({
+    const result = TaggedTypeFormat.encodeInterpolation({
```

Comment:
is the `@object` behavior still tested? From what I can tell, it looks like all these tests just got refactored, and now the `@object` behavior isn't tested.

Reply:
The diff is pretty gnarly now, but I put the original tests back together and re-titled the new ones for `encodeInterpolation`